operate/importer-8_5/src/main/java/io/camunda/operate/zeebeimport/v8_5/processors/ListViewZeebeRecordProcessor.java

```diff
@@ -244,6 +244,9 @@ public void processProcessInstanceRecord(
         logger.debug("Process instance for list view: id {}", piEntity.getId());
 
         if (canOptimizeProcessInstanceIndexing(piEntity)) {
+          // TODO we can't do this anymore if we use scripts
+          // may be actually already collide with incident post importer, when we set incident =
+          // false/true flag
```

Comment:
As we would merge this PR individually of the follow-up ticket, I suggest to leave the TODO out of the code and make a note in the follow-up ticket.